### PR TITLE
Tickets/DM-39057

### PR DESCRIPTION
**Pull request by gmegh**

### CODE REVIEW

File: python/lsst/ts/standardscripts/data/scripts/maintel/prepare_for/align.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/data/scripts/maintel/prepare_for/align.py:21]: missing line:

```
# along with this program. If not, see <https://www.gnu.org/licenses/>.
```

File: python/lsst/ts/standardscripts/maintel/prepare_for/__init__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/maintel/prepare_for/__init__.py:20]: missing line:

```
# along with this program. If not, see <https://www.gnu.org/licenses/>.
```

File: tests/test_maintel_prepare_for_align.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
```
Comment (tribeiro) [inline tests/test_maintel_prepare_for_align.py:20]: missing line:

```
# along with this program. If not, see <https://www.gnu.org/licenses/>.
```

File: python/lsst/ts/standardscripts/maintel/prepare_for/align.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        self.tel_align_rot = config.tel_align_rot
+
+    def set_metadata(self, metadata):
+        metadata.duration = 600.0
+
+    async def run(self):
+        """Run the script."""
+
+        await self.checkpoint("Positioning Telescope.")
+        await self.mtcs.point_azel(
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/maintel/prepare_for/align.py:108]: I think moving the telescope should probably be the last thing done by the script..

File: python/lsst/ts/standardscripts/maintel/prepare_for/align.py [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+            az=self.tel_align_az,
+            el=self.tel_align_el,
+            rot_tel=self.tel_align_rot,
+            wait_dome=False,
+        )
+
+        await self.checkpoint("Closing mirror covers.")
+        await self.mtcs.close_m1_cover()
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/maintel/prepare_for/align.py:116]: I have some concerns about this one. I started a conversation on slack but I don't think we can perform the alignment if the covers are closed.

**Review by tribeiro — COMMENTED** [review]
Hear some very minor comments. I think we need to add some unit tests for the run method, but let's wait until we figure out what we want to do with respect to the mirror covers.

File: python/lsst/ts/standardscripts/maintel/prepare_for/align.py (hunk 5 above)
Comment (gmegh) [reply python/lsst/ts/standardscripts/maintel/prepare_for/align.py]: removed

File: tests/test_maintel_prepare_for_align.py [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+import logging
+import random
+import unittest
+
+from lsst.ts import standardscripts
+from lsst.ts.standardscripts.maintel.prepare_for import PrepareForAlign
+
+random.seed(47)  # for set_random_lsst_dds_partition_prefix
+
+logging.basicConfig()
```
Comment (tribeiro) [inline tests/test_maintel_prepare_for_align.py:31]: you don't need this .

**Review by tribeiro — COMMENTED** [review]
I left one comment in the unit test. Also please, update the version history.

**Review by tribeiro — APPROVED** [review]
Looks good.. We should expand the unit tests in the future though.

**Pull request merged**